package.json

```diff
@@ -15,6 +15,8 @@
   "scripts": {
     "analyze:clean": "rimraf stats.json",
     "preanalyze": "npm run analyze:clean",
+    "analyze:watch": "cross-env NODE_ENV=development ANALYZE=true node server",
+    "analyze:watchProd": "cross-env NODE_ENV=production ANALYZE=true node server",
```

Comment:
I don't very like commans with enviromens

Can we pass NODE_ENV from cli?

Reply:
I'm open to anything!

Reply:
@gretzky I think, we can use `ANALYZE=true yarn start` and `ANALYZE=true yarn start:prod`

And we need advanced-configuration section, such as CRA

https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#advanced-configuration